### PR TITLE
Installation Guide Revamp

### PR DESCRIPTION
**Pull request by rauljordan**
This PR revamps the installation and staking guides to be far more linear and self-contained. It creates three versions of a single page called Install with Script, Install with Docker, Install from Source, which inform the user of requirements, dependencies, and step by step modular guides depending on their OS.

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ module.exports = {
     tagline: 'Ethereum 2.0 implementation written entirely in Go.',
     url: 'https://docs.prylabs.network',
     baseUrl: '/',
-    onBrokenLinks: 'throw',
+    onBrokenLinks: 'warn',
```
Comment (rauljordan) [inline website/docusaurus.config.js:6]: I will revert this once v2 docs are ready

File: website/versioned_docs/version-2.0.0/install/install-with-script.md [hunk 2]
```diff
@@ -128,7 +123,263 @@ mkdir prysm && cd prysm
...
+{"syncing":false}%
+```
+
+### Step 2: Send your validator deposit via the Ethereum validator launchpad
+
+:::danger Ensure You Are Not Being Scammed
+The correct address for the launchpad is https://launchpad.ethereum.org and the only, official validator deposit contract is [0x00000000219ab540356cbb839cbe05303d7705fa](https://etherscan.io/address/0x00000000219ab540356cbb839cbe05303d7705fa). Do not send ETH directly to the contract, and only join by using the eth2 launchpad.
+:::
+
+The [Official Eth2 Launchpad](https://launchpad.ethereum.org/summary) is the easiest way to go through a step-by-step process to deposit your 32 ETH to become a validator. Throughout the process, you'll be asked to generate new validator credentials using the official Ethereum deposit command-line-tool [here](https://github.com/ethereum/eth2.0-deposit-cli). Make sure you use the `mainnet` option when generating keys with the deposit CLI. During the process, you will have generated a `validator_keys` folder under the `eth2.0-deposit-cli` directory. You can import all of your validator keys into Prysm from that folder in the next step.
```
Comment (james-prysm) [inline website/versioned_docs/version-2.0.0/install/install-with-script.md:270]: again for testnet either the updated url or the mention on switching the inside the website ( top right corner) for prater

File: website/versioned_docs/version-2.0.0/install/install-with-bazel.md [hunk 3]
```diff
@@ -78,9 +80,81 @@ bazel build //beacon-chain:beacon-chain --config=release
...
+
+Note: You will be asked to do a one time acknowledgement of our [Terms of Use](https://github.com/prysmaticlabs/prysm/blob/master/TERMS_OF_SERVICE.md). You can also read the legal terms first, then confirm them via a flag using --accept-terms-of-use in both your beacon node and validator.
+
+```text
+bazel run //validator:validator -- accounts import --keys-dir=$HOME/eth2.0-deposit-cli/validator_keys
+```
+
+### Step 4: Run your Prysm validator
+
+Open a second terminal window. Depending on your platform, issue the appropriate command from the examples below to start the validator.
```
[the remark this answers is not in the document]
Comment (rauljordan) [reply website/versioned_docs/version-2.0.0/install/install-with-bazel.md:166]: What do you think about holding off on this until we improve web UI stuff post v2 release?

**Review by james-prysm — APPROVED** [review]
just needs to remove the eth2 now

**Pull request merged**